### PR TITLE
[Bugfix:TAGrading] Fix Grade Inquiry Toggle Button

### LINKED ISSUE #10576: "Grade Inquiry Only" button is broken
**Describe the bug**
The "Grade Inquiry Only" button appears to be broken.  See the screen recording below.

**Expected behavior**
The Grade Inquiry Only button should only show students who have an active grade inquiry.

**Screenshots**

https://github.com/Submitty/Submitty/assets/16820599/03cef255-fdc5-4389-9b71-863c306abda5

### PR DESCRIPTION
**Pull request by williamschen23**
Closes #10576 
### What is the current behavior?
There is a lot of extra code, and the Grade Inquiry Only button for grading is broken.

### What is the new behavior?
The grade inquiry function was being run when there was no display for students yet, so I moved it to the bottom. Additionally, some of the logic and code were extra and backwards so I changed them accordingly.

### CODE REVIEW

File: site/app/templates/grading/electronic/Details.twig [hunk 1]
```diff
@@ -763,25 +765,13 @@
...
-        else{
-            status = 'on';
-        }
-        if(status === 'on') {
-            Cookies.set('inquiry_status', 'on');
-            var inquiry = Cookies.get('inquiry_status');
-            grade_inquiry_only();
-            button.textContent = "Grade Inquiry Only: On";
-        } else {
+        var status = {{ inquiry_status ? 1 : 0}};
```
Comment (zacharymnp) [inline site/app/templates/grading/electronic/Details.twig:768]:
```suggestion
        const status = {{ inquiry_status ? 1 : 0}};
```

File: site/app/templates/grading/electronic/Details.twig [hunk 2]
```diff
@@ -763,25 +765,13 @@
...
+        if(status) {
```
Comment (zacharymnp) [inline site/app/templates/grading/electronic/Details.twig:769]:
```suggestion
        if (status) {
```

File: site/app/templates/grading/electronic/Details.twig [hunk 3]
```diff
@@ -763,25 +765,13 @@
...
             Cookies.set('inquiry_status', 'off');
-            var inquiry = Cookies.get('inquiry_status');
-            reverse_inquiry_only();
-            button.textContent = "Grade Inquiry Only: Off";
+        } else {
```
Comment (zacharymnp) [inline site/app/templates/grading/electronic/Details.twig:771]:
```suggestion
        } 
        else {
```

**Review by zacharymnp — CHANGES_REQUESTED** [review]
There are a few small code style issues in change_inquiry().

**Review by zacharymnp — APPROVED** [review]

File: site/app/templates/grading/electronic/Details.twig [hunk 4]
```diff
@@ -60,7 +60,9 @@
     </div>
     {% if grade_inquiry_only_button %}
         <script>
-            inquiry_update();
+            $(document).ready(function(){
+                inquiry_update();
+            })
```
Comment (williamjallen) [inline site/app/templates/grading/electronic/Details.twig:65]: Would it be better to put the document.ready in the function itself instead?

File: site/app/templates/grading/electronic/Details.twig [hunk 5]
```diff
@@ -764,25 +766,14 @@
...
+        const status = {{ inquiry_status ? 1 : 0}};
+        if (status) {
             Cookies.set('inquiry_status', 'off');
-            var inquiry = Cookies.get('inquiry_status');
-            reverse_inquiry_only();
-            button.textContent = "Grade Inquiry Only: Off";
+        } 
+        else {
+            Cookies.set('inquiry_status', 'on');
         }
```
Comment (williamjallen) [inline site/app/templates/grading/electronic/Details.twig:775]: The status constant isn't needed here.  Can't you just use Twig to conditionally set the value of the cookie?

**Review by williamjallen — CHANGES_REQUESTED** [review]

File: site/app/templates/grading/electronic/Details.twig (hunk 5 above)
Comment (williamschen23) [reply site/app/templates/grading/electronic/Details.twig:775]: It was rather consistent across the many change functions. Would you like me to refactor all of them?

File: site/app/templates/grading/electronic/Details.twig (hunk 4 above)
Comment (williamschen23) [reply site/app/templates/grading/electronic/Details.twig]: I put the inquiry_update in the ending script tag and removed the jquery altogether.

**Review by williamjallen — APPROVED** [review]
Code looks good.

**Pull request merged**